index.html

```diff
@@ -15,5 +15,87 @@
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
 
+<header>
+   
+    <div class = "logo">
+     <img src= img/karma-logo.svg>     
+    </div> 
+        
+    <nav class = "container_box">
+        <ul>
+            <li> Meet Karma   </li>
+            <li class="gray"> How it Works   </li>
+            <li class="gray"> Store   </li>
+            <li class="gray"> Blog  </li>
+            <li class="gray"> Help   </li>
+            <li class="gray"> Login  </li>
+        </ul>
+    </nav>
+</header>
+
+
+<main>
+
+    <div class = "hero">
+
+        <div class="hero-text">   
+        <h1>Introducing Karma</h1>
+        <p>Bring Wifi with you, everywhere you go.</p>
+
+        <btn id="learn_btn">Learn More</btn>
+        </div>
+ 
+    </div>
+
+    <div >
+
+        <div class = "icons_container_text">Everyone needs a little Karma.</div>
+
+        <div class = "icons_container">
+            <div>
+                <img src = "img/icon-devices.svg" alt ="icon devices" class = "icons_images">
+                <p>Internet for all devices</p>
+            </div>
+            <div>
+                <img src = "img/icon-coffee.svg" alt = "coffee mug" class = "icons_images">
+                <p>Boost your productivity</p>
+            </div>
+            <div>
+                <img src= "img/icon-refill.svg" alt = "charging a device" class = "icons_images">
+                <p>Pay as You Go</p>
+            </div>
+
+        </div>
+    </div>
+
+</main>
+
+
+
+<footer>
```

Comment:
Does `footer` need to wrap `div class="footer"`? The fewer elements we can get away with the better!